### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by MSeechan**

### CODE REVIEW

File: notifications/front_end/src/components/Notifications.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <input />
+      <hr />
+      <br />
+      <label>Field 12</label>
+      <input />
+      <br />
+      <label>Field 13</label>
+      <input />
+      <hr />
+
```
Comment (Ryan1Up) [inline notifications/front_end/src/components/Notifications.js:54]: So many fields

File: notifications/front_end/src/components/reports.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export default () => {
+  return (
+    <section>
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Amount Due</th>
+            <th>Date</th>
+            <th>Insurance Co</th>
```
Comment (Ryan1Up) [inline notifications/front_end/src/components/reports.js]: Insurance?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```
Comment (Ryan1Up) [inline .gitignore]: i think you will have to use **/node_modules/** as it doesn't look like they were ignored

Comment (MSeechan) [reply .gitignore]: I thought it worked when the folder looked grayed-out on vsc. Dang. I'll try this again.

**Pull request closed without merging**